src/runtime-rs/crates/resource/src/manager_inner.rs

```diff
@@ -146,6 +146,11 @@ impl ResourceManagerInner {
                         .await
                         .context("do handle vsock device failed.")?;
                 }
+                ResourceConfig::Protection(p) => {
+                    do_handle_device(&self.device_manager, &DeviceConfig::ProtectionDevCfg(p))
+                        .await
+                        .context("do handle vsock device failed.")?;
```

Comment:
It should be `"do handle protection device failed."`

Reply:
Fixed.